packages/web/docs/tailwind.config.ts

```diff
@@ -42,6 +42,7 @@ const config: Config = {
     },
   },
   plugins: [
+    ...baseConfig.plugins,
```

Comment:
The base config includes the container queries plugin.

It's already built-in in Tailwind 4, but I don't want to do that migration yet (ever? before AI can e2e test it looking for visual regressions?)